packages/@okta/vuepress-site/docs/guides/email-magic-links-overview/main/index.md

```diff
@@ -0,0 +1,243 @@
+
+### Same device, same browser only
+
+Magic links only work when there is complete assurance that the person who started the request is the same one who clicked the magic link. For this reason, a user who starts signing in to your application in a web browser must be in the same browser on the same device when they click the magic link. If their browser or device is different, the magic link is disabled. They must either return to the original browser to use the magic link or use the OTP instead. The following flowchart illustrates this logic.
+
+<div class="full">
+
+![Flow diagram showing when and how the magic link flow falls back to the OTP flow](/img/advanced-use-cases/email-magic-links-overview-fallback-to-otp.png)
+
+</div>
+
+This same browser enforcement also ensures that a user doesn't sign in to applications from non-managed devices. For example, should they start the sign-in process on a company laptop, and then click the magic link from their personal device, they must return to the company laptop to complete the process.
+
+### Pros and Cons
+
+When you evaluate whether to support magic links in addition to OTPs in the email authenticator, consider the following:
+
+* A user has a better sign-in experience from magic links than OTP if they sign in from the same browser on the same device.
+* If a user signs in from a different browser or device, the process falls back to using an OTP.
+* A user can only access an application by clicking a magic link on a device that already has access to that application.
+* Okta provides support for both OTP and Magic Links in their Embedded Sign-In Widget and Identity Engine SDKs.
+
+## Integrate Magic Links
+
+Integrating Email Magic Links into your application is a two-step process.
+
+1. Create an endpoint for your magic link. This endpoint checks the validity of the OTP and sends it to Identity Engine for validation.
+1. Set that endpoint as the callback URL for magic links for your application with the Admin Console for your org.
+
+### Create an endpoint for your magic links
+
+Any magic link URL pointing to your application contains two query parameters:
+
+* The one-time password that validates the user (`otp`)
+* A state token that uniquely identifies the current authentication process and its state (`state`)
+
+For example, `http://${yourOktaDomain}?otp=726009&state=1b31fa98b34c45d9a`.
+
+You must create an endpoint for your application that:
+
+1. Retrieves `otp` and `state` values from the query parameters
+2. Matches the state token with the current `state` in your user's browser session
+3. Makes any other checks you deem necessary to ensure that the user is working in the same browser on the same device
+4. Requests that the user enter the `otp` value manually if steps 2 or 3 fail
+5. Sends `otp` and `state` to Identity Engine for validation if steps 2 and 3 pass
```

Comment:
the `state` in step 5 is the Idx API `state handle`. I'm not aware of this concept or the Idx API being documented yet. Step 5 seems confusing as it reads as the `state` is the same as in step 1 and it is not. The SDKs will use the `state` from step 1 to find the idx state handle to be used on step 5.